AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -907,4 +899,15 @@ - (void)rangeController:(ASRangeController *)rangeController didDeleteSectionsAt
   }
 }
 
+#pragma mark - ASCellNodeDelegate
+
+- (void)node:(ASCellNode *)node didRelayoutWithSuggestedAnimation:(ASCellNodeAnimation)animation
+{
+  ASDisplayNodeAssertMainThread();
+  NSIndexPath *indexPath = [self indexPathForNode:node];
+  if (indexPath != nil) {
+    [super reloadItemsAtIndexPaths:@[indexPath]];
```

Comment:
After the changes in #812, it's up to the developer the animate the layout changes in multiple cells, i.e they can wrap multiple `-setNeedsLayout` calls inside a single animation block, or don't animate at all.